cpp/box/Box.h

```diff
@@ -23,6 +26,9 @@ constexpr float TWO_PI = 2.0 * M_PI;
 
 namespace freud { namespace box {
 
+template<typename T>
+using pybind11_array = pybind11::array_t<T, pybind11::array::c_style | pybind11::array::forcecast>;
```

Comment:
Best place for this typedef? I was thinking in freud::util since it will likely be used by each freud module.